src/js/plugins/plugin.search.view.js

```diff
@@ -0,0 +1,317 @@
+class SearchView {
+  constructor(params) {
+    if (!params.selector) {
+      console.warn('BookReader::Search - SearchView must be passed a valid CSS selector');
+      return;
+    }
+
+    this.br = params.br;
+
+    this.matcher = new RegExp('{{{(.+?)}}}', 'g');
+    this.dom = {
+      searchTray: this.renderSearchTray(params.selector),
+    };
+    this.cacheDOMElements();
+    this.bindEvents();
+
+    if (this.br.options.initialSearchTerm) {
+      this.setQuery(params.query);
+    }
+  }
+
+  cacheDOMElements() {
+    this.dom.results = this.dom.searchTray.querySelector('[data-id="results"]');
```

Comment:
Q: are these the full list of search dom?

Reply:
This is the search results container in the search panel.

Reply:
pardon, i meant the full list declaration in `cacheDOMElements`